README.md

```diff
@@ -18,6 +18,7 @@
 ## Requirements
 
 * Mac OS X or Linux
+* Docker 
```

Comment:
We should make this
```
* Docker (for building and running tests)
```
Otherwise it could appear this is a runtime requirement for deployments.

Reply:
> Docker (for building and running tests)

well, Let me change it.